### PR TITLE
Add development package to build

### PR DESCRIPTION
**Pull request by mmitche**
(Note once the other PDB changes go in I'll rebase this onto master to remove some of the noisy diffs from that change)

Adds a "development" package to the build.  This builds a nuget package that contains the PDBs as well as headers that can be used for development of other components.

### CODE REVIEW

**Comment by mmitche** [comment]
Waiting on #96 to finish before this goes in

File: src/inc/CMakeLists.txt [hunk 1]
```diff
@@ -60,3 +60,10 @@ endif(WIN32)
 
 # Compile *_i.c to lib
 add_library(corguids ${CORGUIDS_SOURCES})
+
+# Binplace the inc files for packaging later.
+
+install (FILES cor.h DESTINATION inc)
+install (FILES corhdr.h DESTINATION inc)
+install (FILES corinfo.h DESTINATION inc)
+install (FILES corjit.h DESTINATION inc)
```
Comment (mathstuf) [inline src/inc/CMakeLists.txt]: FWIW, this can be one call:

```cmake
install(FILES core.h corhdr.h corinfo.h corjit.h DESTINATION inc)
```

Comment (mmitche) [inline src/inc/CMakeLists.txt]: Thanks will update

**Comment by mmitche** [comment]
Did anyone take a look at this?

**Comment by mmitche** [comment]
@ellismg maybe you? :D :D

**Comment by ellismg** [comment]
LGTM.

**Pull request merged**